GlobalDefs.h

```diff
@@ -0,0 +1,18 @@
+#ifndef COLORCODERGLOBALDEFS_H

```

Comment:
File name is GlobalDefs, but internally its color-coder global defs.
Possibility to make the file name more specific?
A name like GlobalDefs tends to collect unrelated functionality with successive releases

Reply:
Thanks for pointing it out, I have changed it to ColorCodeDefs.